programs/locked-voter/src/instructions/dispute_instructions/open_dispute.rs

```diff
@@ -0,0 +1,84 @@
+use crate::*;
+/// Accounts for [voter::open_dispute].
+#[derive(Accounts)]
+pub struct OpenDispute<'info> {
+    /// [Escrow].
+    #[account(
+        mut,
+        has_one = owner,
+    )]
+    pub escrow: Box<Account<'info, Escrow>>,
+
+    #[account(mut, constraint = escrow.tokens == escrow_tokens.key())]
+    pub escrow_tokens: Account<'info, TokenAccount>,
+
+    /// [Request].
+    #[account(
+        init,
+        payer = payer,
+        space = 8 + DisputeRequest::LEN
+    )]
+    pub dispute_request: Box<Account<'info, DisputeRequest>>,
```

Comment:
the owner need to open a new dispute, because old dispute is invalid if attacker close_dispute 
If his private key is not leaked, the dispute is still open if they don't close_dispute